packages/common/container-definitions/src/deltas.ts

```diff
@@ -61,21 +61,94 @@ export interface IDeltaSender {
     flush(): void;
 }
 
-/** Events emitted by the Delta Manager */
+/**
+ * Events emitted by {@link IDeltaManager}.
+ */
+/* eslint-disable @typescript-eslint/unified-signatures */
 export interface IDeltaManagerEvents extends IEvent {
+    /**
+     * TODO
+     *
+     * @remarks Listener parameters:
+     *
+     * - `messageBuffer`: TODO
+     */
     (event: "prepareSend", listener: (messageBuffer: any[]) => void);
+
+    /**
+     * TODO
+     *
+     * @remarks Listener parameters:
+     *
+     * - `message`: TODO
+     */
     (event: "submitOp", listener: (message: IDocumentMessage) => void);
+
+    /**
+     * Emitted immediately after processing an incoming operation (op).
+     *
+     * @remarks Listener parameters:
+     *
+     * - `message`: The op that was processed.
+     *
+     * - `processingTime`: TODO
+     *
+     * @internal
+     */
     (event: "op", listener: (message: ISequencedDocumentMessage, processingTime: number) => void);
+
+    /**
+     * @deprecated Please use the "saved" event instead.
+     */
     (event: "allSentOpsAckd", listener: () => void);
-    (event: "pong" | "processTime", listener: (latency: number) => void);
+
+    /**
+     * TODO
+     *
+     * @remarks Listener parameters:
+     *
+     * - `latency`: TODO
+     *
+     * @internal
+     */
+    (event: "pong", listener: (latency: number) => void);
+
     /**
-     * The connect event fires once we've received the connect_document_success message from the
-     * server.  This happens prior to the client's join message (if there is a join message).
+     * @deprecated No replacement API recommended.
```

Comment:
For anything deprecated, we need to make a mark in breaking.md

Reply:
Thank you for the reminder. I will add entries for all newly deprecated items.